### PR TITLE
Web updates needed from CNCF

### PR DESCRIPTION
**Pull request by jonasrosland**
Web updates needed from CNCF, based on https://github.com/goharbor/goharbor.github.io/issues/2.

Signed-off-by: jonasrosland <jrosland@vmware.com>

### CODE REVIEW

File: themes/harbor/layouts/partials/footer.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
           <img src="{{ printf "img/logos/%s" $image | absURL }}">
         </div>
         <div class="footer-copyright">
-          <p>&copy; 2018 The Harbor Authors</p>
+          <p>&copy; 2018 Project Harbor Authors</p>
```
Comment (clouderati) [inline themes/harbor/layouts/partials/footer.html]: Confirmed the CNCF that we should use "The Harbor Authors" to match projects like K8s. Any objections?

Comment (jonasrosland) [reply themes/harbor/layouts/partials/footer.html]: I took the line from @dankohn over at https://github.com/goharbor/goharbor.github.io/issues/2. I can remove this :)

Comment (dankohn) [reply themes/harbor/layouts/partials/footer.html]: We'd recommend:
```html
<p>&copy; Project Harbor Authors</p>
```

Comment (clouderati) [reply themes/harbor/layouts/partials/footer.html]: Sounds good, thanks @dankohn.

**Pull request merged**